### DISCUSSION TITLE
TTY::Link Version 0.1.2?

### DISCUSSION THREAD

**Discussion by capripot** [piotrmurach/tty#89] [Ideas]
Hi there! 
Do you think we could cut the 0.1.2 version of `TTY::Link` including the last changes?

**Comment by piotrmurach** [comment]
Hi 👋

Why do you need a new release? And why the 0.1.2 version in particular? I haven't changed anything yet regarding functionality so a new release wouldn't make any difference. I started refactoring internals to improve maintenance and lay the foundation for adding more terminal checks. This is still a work in progress. The goal is to support more modern terminals as discussed in https://github.com/piotrmurach/tty-link/issues/1. If you are looking for contribution then reading the issue is a first good step. When I finally release a new version it will be 0.2.0.
